perception/bytetrack/include/bytetrack/bytetrack_visualizer_node.hpp

```diff
@@ -0,0 +1,87 @@
+#ifndef BYTETRACK__BYTETRACK_VISUALIZER_NODE_HPP_
```

Comment:
Pls add license

Reply:
@yukkysaito 
Thank you very much for your prompt review!
And I appreciate your pointing this out. I've added and updated license statements in all source files in the modification at 0408d435d61bec6d31fd897b7a0319e8c777d331